### DISCUSSION TITLE
how to display the chess board somewhere else then in the terminal

### DISCUSSION THREAD

**Discussion by SebastiaanDenHertog** [niklasf/python-chess#830] [Q&A]
hi, everyone,

We are working on a school project and we wanna display the chessboard somewhere else than in the terminal. 
Is this possible?

**Comment by niklasf** [comment]
Hi, how about Jupyter notebooks? Example: https://jupyter.brynmawr.edu/services/public/dblank/CS371%20Cognitive%20Science/2016-Fall/Programming%20a%20Chess%20Player.ipynb.
